compiler/extccomp.nim

```diff
@@ -822,6 +822,15 @@ proc linkViaResponseFile(conf: ConfigRef; cmd: string) =
   finally:
     removeFile(linkerArgs)
 
+proc linkViaShellScript(conf: ConfigRef; cmd: string) =
+  let linkerScript = conf.projectName & "_" & "linkerScript.sh"
+  writeFile(linkerScript, cmd)
+  let shell = getEnv("SHELL")
+  try:
+    execLinkCmd(conf, shell & " " & linkerScript)
```

Comment:
What happens here if the project name contains spaces?

Reply:
Cannot happen, the project name is the main Nim module name.

Reply:
@Varriount I also wondered about `linkerScript` ending up with a space in it, but figured it was safe since `let linkerArgs = conf.projectName & "_" & "linkerArgs.txt"` is used in the proc above. If there is a problem with this code, we should also fix the code above it.